html/createUser.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Create User</title>
+</head>
+<body>
+    <ul>
```

Comment:
This `<ul>` may have a better semantic home within a `<nav>` tag

Reply:
Thanks for the recommendations. Implemented a lot of these in Unit 2.2 or 2.3 I believe.